esmvaltool/recipes/examples/recipe_check_obs.yml

```diff
@@ -646,6 +646,13 @@ diagnostics:
          type: reanaly, version: v2, start_year: 1871, end_year: 2012}
     scripts: null
 
+  NOAA-MBL-CH4:
+    description: NOAA marine boundary layer CH4
+    variables:
+      ch4s:
+    additional_datasets:
+      - {dataset: NOAA-MBL-CH4, project: OBS, mip: Amon, type: atmos, version: 1.0, tier: 2}
+    scripts: null
 
```

Comment:
Yes, I am unsure what would happen if the downloaded .csv file covers different years. Would you recommend to add years of the current available period?

Reply:
@valeriupredoi: any recommendations here?

Reply:
I would indeed - it's good to have it, if the user wants only a few years rather than download/cmorize the entire data span. In this case my review below becomes void since I am suggesting removing exactly these optional arguments (start/end times), if they are not used :+1: